roles/matrix-nginx-proxy/templates/nginx/conf.d/matrix-jitsi.conf.j2

```diff
@@ -53,20 +53,25 @@
 
 		tcp_nodelay on;
 	}
-
+	{% if matrix_nginx_proxy_proxy_jitsi_additional_jvbs_enabled %}
+	{% for jvb in matrix_nginx_proxy_proxy_jitsi_additional_jvbs %}
+	{% for id in jvb %}
 	# colibri (JVB) websockets for additional JVBs
-	location ~ ^/colibri-ws/([a-zA-Z0-9-\.]+)/(.*) {
-		proxy_pass http://$1:9090/colibri-ws/$1/$2$is_args$args;
+	location ~ ^/colibri-ws/{{ id }}/(.*) {
+		proxy_pass http://{{ jvb[id] }}:9090/colibri-ws/{{ id }}/$2$is_args$args;
```

Comment:
Why `$2`? I don't see 2 regex groups in the location statement (`location ~ ^/colibri-ws/{{ id }}/(.*) {`).

Reply:
Yeah that's a mistake on my behalf forgot to change it to $1